homeassistant/helpers/service.py

```diff
@@ -955,6 +957,7 @@ def async_register_admin_service(
     ],
     schema: VolSchemaType = vol.Schema({}, extra=vol.PREVENT_EXTRA),
     supports_response: SupportsResponse = SupportsResponse.NONE,
+    description_placeholders: Mapping[str, str] | None = None,
```

Comment:
I don't think there's a test which shows passing `description_placeholders` to `async_register_admin_service` actually works?